python/ucxx/pyproject.toml

```diff
@@ -72,6 +73,7 @@ ninja.make-fallback = true
 sdist.exclude = ["*tests*"]
 sdist.reproducible = true
 wheel.packages = ["ucxx"]
+wheel.exclude = ["*.pyx", "*.pxd", "CMakeLists.txt", "CPPLINT.cfg", ".clang-format"]
```

Comment:
I'm not sure if I understand why we have `.clang-format` and `CPPLINT.cfg` files in `python/ucxx/ucxx/examples/`. Maybe @pentschev knows? If we can rely on the clang-format config in the root of the repo, that would be better imo.

Reply:
Those files were introduced in https://github.com/rapidsai/ucxx/pull/46. My guess is they were copied-and-pasted from somewhere else by mistake, because that PR did not add any C++ files to this directory. I'll remove them and take them out of `wheel.exclude`.